PulseAudio muted volume support

Watcom opened the pull request:
PulseAudio muted volume support, which was missing in the original implementation.

stapelberg commented on src/print_volume.c:
```diff
@@ -67,17 +67,29 @@ void print_volume(yajl_gen json_gen, char *buffer, const char *fmt, const char *
         if (ivolume >= 0) {
-            outwalk = apply_volume_format(fmt, outwalk, ivolume);
+            if (muted)
+                START_COLOR("color_degraded");
```
Where’s the corresponding `END_COLOR`? For some output formats (e.g. xmobar), that’s important.

stapelberg commented:
Please remove “(resolves #27)” from the title and add “fixes #27” to the commit description to trigger github’s hooks.

Watcom commented:
I think "resolves" also triggers the hooks (according to their documentation anyway), but I changed to "fixes" to make it more consistent.

Fixed the missing END_COLOR by setting pbval to 0.
